ra-optimisation/search-tree-rao/src/test/java/com.farao_community.farao.search_tree_rao/process/search_tree/TreeTest.java

```diff
@@ -47,8 +49,16 @@ public void buildOutputTest() {
         Mockito.when(leafOptimal.getNetworkActions()).thenReturn(Collections.singletonList(na));
         Mockito.when(leafOptimal.getCost()).thenReturn(0.0);
 
+        // Mock Crac
+        Crac crac = Mockito.mock(Crac.class);
+        RangeAction rangeAction = Mockito.mock(RangeAction.class);
+        String rangeActionId = "PRA_PST_BE";
+        Mockito.when(crac.getRangeActions()).thenReturn(Collections.singleton(rangeAction));
+        Mockito.when(rangeAction.getId()).thenReturn(rangeActionId);
+        Mockito.when(rangeAction.getName()).thenReturn(rangeActionId);
```

Comment:
maybe load a json crac to avoid too many mocks.
For instance the small-crac.json (in the tests of the LinearRao) already includes a range action with this same id.

Reply:
or maybe this is not needed anymore. I'm not sure we need the crac in the buildOutput function if we take the range actions from the optimalLeaf instead of taking them from the crac.

Reply:
indeed, not needed anymore!